drivers/serial/uart_mcux_lpuart.c

```diff
@@ -30,10 +30,10 @@ LOG_MODULE_REGISTER(uart_mcux_lpuart, LOG_LEVEL_ERR);
 
 #define PINCTRL_STATE_FLOWCONTROL PINCTRL_STATE_PRIV_START
 
-#if defined(CONFIG_UART_LINE_CTRL) &&  \
-	defined(FSL_FEATURE_LPUART_HAS_MODEM_SUPPORT) && \
-	(FSL_FEATURE_LPUART_HAS_MODEM_SUPPORT)
-#define UART_LINE_CTRL_ENABLE
+/* Helper macros */
+#if defined(FSL_FEATURE_LPUART_HAS_MODEM_SUPPORT) && \
+	FSL_FEATURE_LPUART_HAS_MODEM_SUPPORT
+#define LPUART_HAS_MODEM
```

Comment:
let's change this to be defined to 1, so that we can use IS_ENABLED

Reply:
done, thank you